sbnana/CAFAna/Core/Spectrum.cxx

```diff
@@ -63,6 +63,57 @@ namespace ana
     loader.AddSpectrum(*this, var, cut, wei);
   }
 
+  //----------------------------------------------------------------------
+  Spectrum::Spectrum(const std::string& label, const Binning& bins,
+                     SpectrumLoaderBase& loader,
+                     const TruthVar& var,
+                     const TruthCut& truthcut,
+                     const SpillCut& spillcut,
```

Comment:
Maybe the shift has already sailed on this one due to people using this in branches as we've gone, but I wonder if it would be somehow "cleaner" to do SpillCut then TruthCut in parallel with the above being SpillVar then (Slice) Var, and similar comment could apply below.

Anyway, I really don't think this matters too much.

Reply:
Ah actually looking even further below I think I see the logic is more like TruthCuts then "reco cuts" where SpillCut is kind-of in-between. This is fine.